Css frameworks

elli95 opened the pull request.

elli95 committed:
First version of cc frameworks ca

First version of the cc framework task.
Should contain everything from html, css/scss and bootstrap that is necessary for the task.
Must be evaluated.

elli95 committed:
Added dist/css/ folder with ccs files

Added dist/css/ folder with ccs files

sulenchy commented on index.html:
```diff
@@ -0,0 +1,34 @@
+                </div>
+                <div class="d-flex flex-column">
+                    <label for="password"  class="form-label">Password</label>
+                    <input type="text" name="password" id="password" class="form-control" minlength="8"  placeholder="Password" required>
```
This should be of type `password`

elli95 replied:
Thank you!
I will fix this.

sulenchy commented on feed/index.html:
```diff
@@ -0,0 +1,105 @@
+        </section>
+        <!-- old post -->
+        <section class="container mt-4 d-flex flex-column col-sm-10 col-lg-7 col-xxl-4">
+            <ul class="list-group gap-2">
```
On a bigger screen, you have the thumbnails stacked up on a column. It would be better if you could utilize the spaces around the thumbnails on a bigger screen. Try this 
```
<ul class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3 gap-1">
...
</ul>```

You can also find an example at https://getbootstrap.com/docs/5.3/examples/album/.

elli95 replied:
Thanks for the reply, didn't think much of it at the moment.
I will adjust it.

sulenchy reviewed (changes requested):
Have a few comments for you at the top.

elli95 committed:
Changed type from text to password

Changed type from text to password

elli95 committed:
Change layout on post list

Switch from simple colonne to changing colonne from 1 to 3 based on screen size.

elli95 committed:
LiveServer setting port
